### PR TITLE
add fastify example

### PR DESCRIPTION
**Pull request by Ethan-Arrowood**
This adds an example of using now to deploy [fastify](https://fastify.io)

### CODE REVIEW

**Comment by msweeneydev** [comment]
Hey @Ethan-Arrowood, thank you for contributing this! To enhance this example further, could you break it out into two folders, `/api` with your current content and `/www` with a static frontend? You can use this example to get an idea of how that would look - https://github.com/zeit/now-examples/tree/master/nodejs

**Comment by Ethan-Arrowood** [comment]
It seems to be failing because of a dependency? Any ideas? 
https://zeit.co/ethan-arrowood/now-fastify/f7lk65wd3/logs

File: now.json [hunk 1]
```diff
@@ -94,6 +94,7 @@
       "config": { "distDir": "build" }
     },
     { "src": "nextjs-mysql/api/**/*.js", "use": "@now/node" },
-    { "src": "nextjs-mysql/next.config.js", "use": "@now/next" }
+    { "src": "nextjs-mysql/next.config.js", "use": "@now/next" },
+    { "src": "fastify/index.js", "use": "@now/node-server" }
```
Comment (msweeneydev) [inline now.json]: @Ethan-Arrowood this looks to be the issue as there is no `/fastify/index.js` file, you will need to add a builder for the static part as well.

**Comment by msweeneydev** [comment]
@Ethan-Arrowood I've added a comment to show where the problem is, you should add a builder in the root `now.json` for the static part as well.

This is looking really good, just let me know if I can help further!

**Comment by msweeneydev** [comment]
Hey @Ethan-Arrowood, could you take a look at the requested changes please? After those it should be good to go 👍

**Comment by Ethan-Arrowood** [comment]
Hi Yes sorry about that. got hit with some midterm exams and projects that have taken up all my time until now. Will finish this soon!

**Comment by msweeneydev** [comment]
Not to worry @Ethan-Arrowood, it'll be here when you're ready 👍

**Comment by msweeneydev** [comment]
Hey @Ethan-Arrowood, we've recently had a change in the direction of this repository based on customer feedback. We are now looking to provide examples based around frontends, rather than serverful frameworks. As a result, I am closing this PR.

Thank you for taking the time to contribute, it is much appreciated.

**Pull request closed without merging**